src/components/helpers/chocolate_sauce.ts

```diff
@@ -1,22 +1,24 @@
 import urlMetadata from 'url-metadata';
 import * as htmlparser2 from 'htmlparser2';
 import fetch from 'node-fetch';
+import {decode} from 'html-entities';
 
 
 async function fetchUrl(url: string) {
   try {
 	const response = await fetch(url, { headers: {'User-Agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/114.0.0.0 Safari/537.36'}});
 	const body = await response.text();
+	console.log("body is " + body)
```

Comment:
I say remove this since you know it's working.

Reply:
def will eventually but its really helpful debugging specific rss feeds that behave stupidly